utilities/options/options_util_test.cc

```diff
@@ -554,65 +554,114 @@ TEST_F(OptionsUtilTest, BadLatestOptions) {
   s = LoadLatestOptions(config_opts, dbname_, &db_opts, &cf_descs);
   ASSERT_NOK(s);
   ASSERT_TRUE(s.IsInvalidArgument());
+
+  // We removed a parameter of the lower version in this PR:
+  // https://github.com/tikv/rocksdb/pull/286, so the logic
+  // checked below is not valid. We've decide to turn off
+  // checking for now until a better way is found, so just
+  // skip this test.
+#if 0
```

Comment:
Let's define a special macro in this file instead of copying comment many times. `#ifndef NO_FORCE_UNKNOWN_OPTIONS_CHECK`

Reply:
Fixed, thanks!